README.md

```diff
@@ -202,7 +210,7 @@ request()
   .catch();
 ```
 
-:rotating_light: *_PLEASE NOTE_*: If you choose the `deferred` option, the SDK will _not_ do any filtering or sorting on the messages internally. You will get the messages exactly as they come down from the API, untouched. This means you may (for example) show in-app messages marked `read` or show the messages in the wrong order based on `priority`.
+:rotating*light: \*\_PLEASE NOTE*\*: If you choose the `deferred` option, the SDK will _not_ do any filtering or sorting on the messages internally. You will get the messages exactly as they come down from the API, untouched. This means you may (for example) show in-app messages marked `read` or show the messages in the wrong order based on `priority`.
```

Comment:
This is causing issue with the emoji, could you take a look?

README from main
![Screenshot 2024-05-23 at 11 56 32 AM](https://github.com/Iterable/iterable-web-sdk/assets/64097726/6e131ca7-d154-45c6-b05a-84ce63ea7ef6)

README from this branch
![Screenshot 2024-05-23 at 11 56 17 AM](https://github.com/Iterable/iterable-web-sdk/assets/64097726/4e12f164-2cae-4b14-87a3-bf5b8cdd1269)

Reply:
weird, must be the prettier formatter... fixed.